nautobot/core/settings.py

```diff
@@ -632,12 +689,14 @@
 CONSTANCE_CONFIG_FIELDSETS = {
     "Banners": ["BANNER_LOGIN", "BANNER_TOP", "BANNER_BOTTOM"],
     "Change Logging": ["CHANGELOG_RETENTION"],
-    "Device Connectivity": ["PREFER_IPV4"],
     "Natural Keys": ["DEVICE_NAME_AS_NATURAL_KEY", "LOCATION_NAME_AS_NATURAL_KEY"],
+    "Device Connectivity": ["NETWORK_DRIVERS", "PREFER_IPV4"],
+    "Installation Metrics": ["DEPLOYMENT_ID"],
```

Comment:
Looks like we were trying to keep these fieldsets in alphabetical order previously:

```suggestion
    "Device Connectivity": ["NETWORK_DRIVERS", "PREFER_IPV4"],
    "Installation Metrics": ["DEPLOYMENT_ID"],
    "Natural Keys": ["DEVICE_NAME_AS_NATURAL_KEY", "LOCATION_NAME_AS_NATURAL_KEY"],
```